docs/guide/seo.rst

```diff
@@ -0,0 +1,63 @@
+==================================================
+Search engine optimization (SEO) for documentation
+==================================================
+
+What is search engine optimization?
+-----------------------------------
+
+Search engines drive a significant portion of site traffic for docs that publish as public-facing websites.
+Search engine optimization (SEO) describes the methods used to improve the rank of your web pages in search results for a given set of search terms.
+
+To perform SEO, you must be familiar with the ever-changing algorithms that search engines use to select and arrange the webpages returned for a given set of search terms.
+These algorithms change frequently, so the requirements for achieving the best optimization also change frequently.
+
+SEO for documentation
+---------------------
+
+Good tech writing follows most of the standard practices required for excellent SEO:
+
+- Write imperative, user-centric, goal-oriented headings
+- Make your docs skimmable by breaking content into sections under appropriate headings
+- Focus on good information architecture that provides quality navigation and site hierarchy
+- Include page metadata, such as last modified date, in the `<head>` section of your web page (how to do this varies by your publication toolchain)
```

Comment:
I believe this is the link causing the issue. To avoid thinking this is a link, you need to double escape it. I think
```suggestion
- Include page metadata, such as last modified date, in the ``<head>`` section of your web page (how to do this varies by your publication toolchain)
```

Reply:
AHHHHHHHHHHHHHHH
Thank you for your eyes.